scripts/qa_crowbarsetup-test.sh

```diff
@@ -2,7 +2,7 @@
 
 describe "roundup(1) testing of qa_crowbarsetup.sh"
 
-verlist="3 4 5 5minus 8 4plus 8M4plus 8M4minus"
+verlist="3 4 5 5minus 9 4plus 9M4plus 9M4minus"
```

Comment:
are 3, 4, 5, 5minus and 4plus version of cloud? if so can we remove them?

Reply:
They meant SOC3/4/5 but are used here only for testing the cloudver comparison (< = >) checks and should not be updated for every new version